### ISSUE TITLE
Question about code 

### ISSUE THREAD

**Issue by Youskrpig** [lukasruff/Classification-AD#2]

**Comment by Youskrpig** [comment]
Hi, thanks for your great work. 
I run the  mnist Experiment(with OE) (normal class:0，abnormal classs: the rest) , OE dataset: emnist (26 classes all)
but in the code , i found in the minibatch: 
 the label and semi-target of mnist training data are 0 and 0;
 the label and semi-target of  emnist training data are 1 and -1;
But in the paper section 3,  "Here y =1 denotes a nominal data point and y=0 denotes an anomaly."
